### ISSUE TITLE
Can not append distributed dataframes

### ISSUE THREAD

**Issue by Norhk** [dask/distributed#2308]
As the title says. Then appening distributed dataframes I get an error:

(The dataframe is distributed on a number of workers.)

ValueError: All inputs have known divisions which cannot be concatenated in order. Specify interleave_partitions=True to ignore order

Passing the argument yields unknown argument error.

dask version '0.19.4'

**Comment by mrocklin** [comment]
It looks like append hard-codes `interleave_partitions` while `concat` allows it. 

@Norhk short term I recommend using concat

@TomAugspurger are there reasons not to allow interleaving when using `append`?

@Norhk if not would you be interested in submitting a small PR to fix?  This should be fairly easy.

**Comment by Norhk** [comment]
Yes I will make a PR, give me 2 weeks (conference...). I will also test this further. The project is amazing!
